com.github.fabrialberio.pinapp.yml

```diff
@@ -0,0 +1,41 @@
+id: com.github.fabrialberio.pinapp
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "43"
+command: pinapp
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+
+  - --env=XDG_DATA_DIRS=/var/lib/flatpak/exports/share:/usr/share
+
+    # Icons
+  - --filesystem=xdg-data/icons:ro
+
+    # Applications
+  - --filesystem=xdg-data/applications:rw
```

Comment:
Generally speaking, it should use Dynamic Launcher portal instead of writing files directly.

Reply:
thanks for the suggestion, I'll look into it